webapp/channels/src/components/profile_popover/profile_popover.scss

```diff
@@ -103,8 +103,8 @@
             padding-inline-start: 16px;
             text-align: center;
 
-            > h5 {
-                margin:0;
+            >h5 {
+                margin: 0;
```

Comment:
even this as well

Reply:
I believe these arse valid changes via the lint fixer, as there should be a space between attribute and value., and for previous one, there shouldn't be space between `>` and selector, so changes are valid, even if unrelated to the PR.